### DISCUSSION TITLE
Copilot stops suggesting when censored words are in code.

### DISCUSSION THREAD

**Discussion by Wiblz** [community/community#10889] [Copilot Conversations]
I am currently working on an RPG game and some parts of my code contain names that include word "race" in it.
Copilot completely stops any suggestions in a file that contains censored words. Removing that part of code instantly fixes the issue.

While I understand the reasoning for this behavior, I would suggest at least adding visual indication and some explanation text (similarly to how network issues are currently handled). Ideally parts of code that have some "dangerous" words should be removed from context instead of completely blocking all the copilot output.

**Comment by rafageist** [comment]
Stupid behavior of Github Copilot. Why?

- Because say a word that offends, does not mean that the person is being offended. 

- And because is a robot, not a human.

Why don't Github not let me choose?

**Reply by gusbemacbe** [reply]
I recommend you to try CodeGeeX. 

See the news: https://www.theregister.com/2021/09/02/github_copilot_banned_words_cracked/

Because GitHub Copilot developers are Microsoft employees, and Microsoft follows the political correctness policy, and as a result, they are required to follow its policy. They treat us like babies and children. 

It was not exclusive to Microsoft. Several big tech companies follow this policy as well.
